website/versioned_sidebars/version-13.0-sidebars.json

```diff
@@ -37,7 +37,6 @@
       "version-13.0-patching/patching-new-fixtures-or-dimmers",
       "version-13.0-patching/changing-the-patch",
       "version-13.0-patching/copying-moving-and-deleting-fixtures",
-      "version-13.0-patching/fixture-personalities",
```

Comment:
Why has this changed?

Reply:
Looks like it was edited in this commit https://github.com/AvolitesLtd/TitanManual/commit/96d0ddf08ef9371e7a0648a3a77d78d9ab5838c0

Reply:
Looks like this was indeed incorrect before in v13